scapy/tools/packet_viewer/viewer.py

```diff
@@ -0,0 +1,126 @@
+import re
+from itertools import count
+from typing import Optional, Union, Iterable, List, Tuple, Callable
+
+import urwid
+from urwid import MainLoop
+
+from scapy.config import conf
+from scapy.packet import Packet_metaclass, Packet, Raw
+from scapy.supersocket import SuperSocket
+from scapy.themes import BlackAndWhite
+from scapy.tools.packet_viewer.main_window import MainWindow
+
+_time = None
+
+
+def viewer(
+        source,  # type: Union[SuperSocket, Iterable[Packet]]
+        columns=None,  # type: Optional[List[Tuple[str, int, Callable]]]
+        basecls=None,  # type: Optional[Packet_metaclass]
+        **kwargs_for_sniff
+):
+    palette = [
+        ("header", "black", "dark green"),
+        ("row_focused", "black", "dark cyan"),
+        ("green", "black", "dark green"),
+        ("red", "white", "dark red"),
+        ("default_bold", "bold", ""),
+    ]
+
+    cf = conf.color_theme
+    conf.color_theme = BlackAndWhite()
+    try:
+        basecls = basecls or getattr(source, "basecls", Raw)
+        columns = columns or get_all_columns(basecls)
+
+        main_window = MainWindow(source, columns, basecls, **kwargs_for_sniff)
+
+        # main_window is the top most widget used to render the whole screen
+        from platform import platform
+        is_wsl = "Microsoft" in platform()
```

Comment:
```suggestion
        is_wsl = "microsoft" in platform().lower()
```
It's `'Linux-4.19.104-microsoft-standard-x86_64-with-Ubuntu-18.04-bionic'`